Recruit-Training/Basic-Recruit-Training/person/Person.cpp

```diff
@@ -0,0 +1,58 @@
+//Solved by Arion Hamel
+#include "Person.h"
+#include <iostream>
+#include <string>
+
+// -- Constructors --
+Person::Person() {
+    name_ = "";
+    age_ = new int(0);
+}
+
+Person::Person(const std::string& name, int age) {
+    name_ = name;
+    age_ = new int(age);
+}
+
+// -- Descontructor --
+Person::~Person() {
+   name_ = "";
+    *age_ = 0;

```

Comment:
Instead of setting age_ to zero, you should use free() instead to deallocate the memory.
[free() guide](https://www.programiz.com/cpp-programming/library-function/cstdlib/free#:~:text=for%20further%20allocations.-,The%20free()%20function%20in%20C%2B%2B%20deallocates%20a%20block%20of,to%20the%20same%20memory%20location.)